source/_integrations/imap.markdown

```diff
@@ -63,6 +63,16 @@ The SSL cipher list and verify SSL are advanced settings. The options are availa
 
 </div>
 
+### Enable IMAP-Push
+
+IMAP-Push is enabled by default if your IMAP server supports it. If you are using an unreliable imap service that drops the connection periodically and causing issues, you might consider to turn off IMAP-Push. This will fallback to polling the IMAP server.
```

Comment:
```suggestion
IMAP-Push is enabled by default if your IMAP server supports it. If you use an unreliable IMAP service that periodically drops the connection and causes issues, you might consider turning off IMAP-Push. This will fall back to polling the IMAP server.
```